controllers/cart.ts

```diff
@@ -66,6 +76,9 @@ export const calculateCartController = buildEndpointRegister({
           finalPrice: Math.round(item.price * discount * 1e2) / 1e2,
         })),
       ],
+      // CON: Improve the precision of the total value. On e-commerce we usually
```

Comment:
Agree. I added a logic for this in L76 and should've replicated in the other places where we're working with prices.

One thing to note is that we would also be using some currency formatter in the client to display the data, e.g
```js
new Intl.NumberFormat('ja-JP', { style: 'currency', currency: 'JPY' }).format(
    number,
  ),
```
which would also strip out the uneeded decimals.